src/SlackMessage.js

```diff
@@ -79,26 +79,23 @@ class SlackMessage extends SlackMessageRoot {
       linkObj = { url: link, text: link };
     }
 
-    const response = await this.#_slackGateway.fetchMessage(
+    const message = await this.#_slackGateway.fetchMessage(
       this.#_channel,
       this.#_ts
     );
 
-    if (response.body.ok) {
```

Comment:
This check isn't useful because the http lib i'm using will throw when the body isn't ok so it's safe to assume if we get to this code that the request was successful